sbi/inference/base.py

```diff
@@ -125,22 +121,29 @@ def __init__(
 
         self._device = configure_default_device(device)
 
-        self._simulator, self._prior = simulator, prior
+        if unused_args.keys():
+            warn(
+                f"You passed some keyword arguments that will not be used. "
+                f"Specifically, the unused arguments are: {list(unused_args.keys())}. "
+                f"These arguments might have been supported in sbi "
+                f"versions <0.14.0. Since 0.14.0, the API was changed. Please consult "
+                f"the corresponding pull request on github: "
+                f"https://github.com/mackelab/sbi/pull/378 and tutorials: "
+                f"https://www.mackelab.org/sbi/tutorial/02_flexible_interface/ for "
+                f"further information.",
```

Comment:
I would suggest adding additional information on how to upgrade/change the code in the changelog as commented above, perhaps even an example of old and new interface side-by-side any directly link to https://github.com/mackelab/sbi/releases/tag/v0.14.0 from here